### PR TITLE
Realistic library simulation

### PR DESCRIPTION
**Pull request by mathemancer**
This starts from a real set of info about some books (provided by @kgodey ) . The starting data set is in `real_books.tsv` . It then cleans that up a bit, and bases a simulation off of the resulting data which adds further editions of those books, as well as copies.

### CODE REVIEW

**Comment by kgodey** [comment]
The data looks mostly fine to me. Some comments:

- Please rename `Publications` to `Books` since that's what we're using for the demo video (it is less precise but more layperson-friendly and doesn't look as much like `Publishers`)
- You're probably already tracking this, but the schema name should be fixed, and we should remove the `real_books_sim` and `real_books_sim_clean` tables.
- When I loaded the SQL file you sent me, it did not create a `Patrons` table and it did not create any FK constraints from `Publications` to `Authors` and `Publishers`.
- I would tone down the number of editions of each book. I think it's fine if the occasional book has multiple editions, but most of them should not. It makes the data in the `Publications` table look un-normalized unless you look very closely.
- Similarly, I think we also want fewer copies of the same book in the `Items` table so that the data looks quite distinct by default.

**Comment by mathemancer** [comment]
These are great notes, thanks.

**Comment by kgodey** [comment]
Distribution looks good!

Comments for both the live demo data set & the demo video data sets:
- Author first name should be to the left of Author last name 

These comments apply only to the demo video files and NOT the main data set for the live demo
- Let's drop a few columns so it's less busy – I would suggest Page Count, Media, both Dewey columns, and Languages. Media and Languages also look de-normalized, so that's a second reason for dropping them. 

Only for live demo data set
- Normalize Media and Languages or drop them

**Comment by mathemancer** [comment]
@kgodey 

Regarding dashes: It seems there isn't a consistent way to do this.  See the table [on Wikipedia](https://en.wikipedia.org/wiki/ISBN) . I propose we skip that for now.

Regarding Author first name vs. last name placement. I already have first name on the left in "Authors" and "Patrons". Am I missing something?

File: realistic_library_simulation/sql_files/simulation.sql
Comment (kgodey) [inline realistic_library_simulation/sql_files/simulation.sql:223]: Please name this column "Book" instead of "Publication" @mathemancer

**Review by kgodey — CHANGES_REQUESTED** [review]

File: realistic_library_simulation/sql_files/simulation.sql
Comment (mathemancer) [reply realistic_library_simulation/sql_files/simulation.sql:223]: Okay, done.

**Comment by mathemancer** [comment]
@kgodey I think we can merge this if you approve (given that the output is already merged in the main repo).

**Review by kgodey — APPROVED** [review]

**Comment by kgodey** [comment]
I did not review any further before merge, I assume that's fine since I reviewed the data set in Mathesar.

**Pull request merged**
